lsp/src/handle.rs

```diff
@@ -68,6 +68,10 @@ fn handle_completion(req: Request) -> Option<HtmxResult> {
         Some(CompletionContext {
             trigger_kind: CompletionTriggerKind::TRIGGER_CHARACTER,
             ..
+        })
+        | Some(CompletionContext {
+            trigger_kind: CompletionTriggerKind::INVOKED,
+            ..
```

Comment:
Looks like VSC*de sends `INVOKED` instead of `TRIGGER_CHARACTER` for HTML `-`'s

Related: microsoft/vscode#177094